File: bidrag-commons-test/src/main/kotlin/no/nav/bidrag/commons/testdatagenerering/adresse/Adressetilknytning.java
package no.nav.bidrag.commons.testdatagenerering.adresse;

import lombok.Data;
import no.nav.bidrag.commons.testdatagenerering.PeriodisertTestData;

import java.time.LocalDate;

@Data
public class Adressetilknytning implements PeriodisertTestData {
    private final TestAdresse adresse;
    private final Adressetype type;
    private final LocalDate periodeFra;
    private final LocalDate periodeTil;
}

File: bidrag-generer-testdata/src/main/kotlin/no.nav.bidrag.generer.testdata/adresse/Adressetilknytning.kt
package no.nav.bidrag.generer.testdata.adresse

import no.nav.bidrag.domene.enums.adresse.Adressetype
import no.nav.bidrag.generer.testdata.PeriodisertTestData
import java.time.LocalDate

@Suppress("unused")
data class Adressetilknytning(
    val adresse: TestAdresse? = null,
    val type: Adressetype? = null,
    override val periodeFra: LocalDate? = null,
    override val periodeTil: LocalDate? = null,
) : PeriodisertTestData
